src/App.js

```diff
@@ -38,7 +39,7 @@ function App() {
       )}
 
       <Routes>
-        <Route path="/" element={<Home />} />
+        <Route path={ROUTE_ROOT} element={<Home />} />
```

Comment:
I think it does not decrease the code readability, as we can easily refer what are the routes that are present in an app through their route constant file.

And using constants for routes make it easy to do changes in route. Suppose right now the route for home page is `/home`, and after some time I want to change it into `/app/home`, if I will not maintain a single source of truth for the routes then, I have to check all the pages, all the components where I am using the route `/home` and then have to make changes their.

By using a constant, I can easily do the changes at only one place and it will be changed everywhere I am using the respective constant.

Using variable for the route `/` may not be obvious because we usually not use it at many place and also we do not change it. But I want to keep consistency, so that's why I used.

Reply:
Thanks for sharing your thoughts. And if you can refer to some blog which says this way is not good then it would be nice. I would definitely give it a read.

Reply:
Ok I got your thoughts.